gitlab/resource_gitlab_repository_file_test.go

```diff
@@ -0,0 +1,357 @@
+package gitlab
+
```

Comment:
could you add some test for `start_branch` attribitue change? I am not sure if it will not require forcing recreation, we should find this out in the tests.

Reply:
there actually is a skipped one here: https://github.com/gitlabhq/terraform-provider-gitlab/pull/724/files#diff-939af460cb608f4acf20ee9378fd6898591ec37c96e4ae94bf494c308beb5dd0R59

However, `start_branch` doesn't seem to work properly, see the description of this PR ...

Reply:
I've enabled the tests.